blog/2024-06-11-automap.md

```diff
@@ -0,0 +1,567 @@
+---
+title: AUTOMAP: How to do NumPy-style broadcasting in Futhark (but better)
+author: Robert Schenck
+---
+
+A lot of code for doing mathematical operations on vectors/arrays involves
```

Comment:
I suggest writing up front that it is you, Robert, who is writing this. The author field is shown, but not very prominently.

Reply:
I added some background info and also a little ad in case someone might know of a postdoc position (if you think this isn't appropriate, I'll remove it).